README.md

```diff
@@ -121,3 +134,23 @@ its configuration and operation.
   [`tokio-console`](https://github.com/tokio-rs/console). See the
   [documentation](docs/CONFIGURING_TOKIO_CONSOLE.md) for configuration
   instructions.
+
+## Versioning and Stability
+
+Janus follows [semantic versioning](https://semver.org/). Because we are at major
+version 0, we increment the minor version number for breaking changes, and
+increment the patch version for new features and backwards-compatible bug fixes.
+
+What is considered a breaking change depends on the crate. The following crates
+are stable on their Rust API:
+- `janus_collector`
+- `janus_client`
+- `janus_messages`
+- `janus_core`
+
+The following crates are stable on their external configuration and HTTP API.
```

Comment:
IMO, mention that these are unpublished (which helps justify the API instability)